client/components/ExampleComponent/CoolComponent.test.js

```diff
@@ -5,3 +5,9 @@ describe('tests can be right next to the component if we add the filepath to the
     expect(true).toBe(true);
   });
 });
+
```

Comment:
wanted to add comment on 3, but it won't let me. Anyway, jest should look for any file in the repo with the word test in the file name, so I think tests can go anywhere